servers/camera_server.cpp

```diff
@@ -42,6 +42,7 @@ void CameraServer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_monitoring_feeds", "is_monitoring_feeds"), &CameraServer::set_monitoring_feeds);
 	ClassDB::bind_method(D_METHOD("is_monitoring_feeds"), &CameraServer::is_monitoring_feeds);
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "monitoring_feeds"), "set_monitoring_feeds", "is_monitoring_feeds");
+	ADD_PROPERTY_DEFAULT("monitoring_feeds", false);
```

Comment:
Is this necessary? The editor should be able to pick up property defaults based on the initial value when the class is created. And `CameraServer.xml` already seems to correctly have the default as `false`

Reply:
Yes, CI failed without it. One of the Linux runners had it default to `true`.

Reply:
The problem is likely that when generating the documentation with `--doctool`, all documented classes are instantiated to get their default property values, which means that the constructor of `CameraTexture` will run and enable monitoring.

It might be preferable to defer it somehow, so that instantiated CameraTextures only register an intent to enable monitoring in CameraServer, and CameraServer does it on the next frame?

Or we just live with this small hack, neither solution is great. It feels a bit weird to have a texture type responsible for enabling a server feature on instantiation.

Reply:
>  It feels a bit weird to have a texture type responsible for enabling a server feature on instantiation.

Yea, that's why I initially wanted to have it be 'just a hack' rather than a feature.
It would be better if Godot dropdowns somehow knew they are about to create a camera texture and enable the monitoring then, but that would require pretty substantial rewrites...